doc/source/release-notes/Percona-Server-5.7.21-21.rst

```diff
@@ -0,0 +1,137 @@
+  unencrypted tables, or ``OFF``, restoring the like-before behavior. Bug fixed
+  :psbug:`1525`.
+
+* Query response time plugin now can be disabled at session level.
+
+Bugs Fixed
+==========
+
+* Query response time plugin installation was able to cause server crash.
+  Bug fixed :psbug:`3959`.
+
+* There was a server crash caused by a materialized temporary table from
+  semi-join optimization with key length larger than 1000 bytes. Bug fixed
+  :psbug:`296`.
+
+* A regression was causing integer overflow with
+  :variable:`thread_pool_stall_limit` variable values bigger than 2 seconds.
+  Bug fixed :psbug:`1095`.
+
+* A memory leak took place in |Percona Server| when performance schema is used
+  in conjunction with thread pooling. Bug fixed :psbug:`1096`.
+
+* A code clean-up was done to fix compilation with clang, as general warnings 
+  (bug fixed :psbug:`3814`, upstream :mysqlbug:`89646`) so clang 6 specific
+  warnings and errors (bug fixed :psbug:`3893`, upstream :mysqlbug:`98111`).
+
+* Compilation warning was fixed for statically linked QRT. Bug fixed
+  :psbug:`3841`.
+
+* |Percona Server| returned empty result for ``SELECT`` query if number of
+  connections exceeded 65535. Bug fixed :psbug:`314`.
+
+* A clean-up in |Percona Server| binlog-related code was made to avoid
+  uninitialized memory comparison. Bug fixed :psbug:`3925` (upstream
+  :mysqlbug:`90238`).
+
+* ``mysqldump`` utility with ``--innodb-optimize-keys`` option was incorrectly
+  working with foreign keys on the same table, producing invalid SQL
+  statements. Bugs fixed :psbug:`1125` and :psbug:`3863`.
+
+* A fix of the mysqld startup script failed to detect jemalloc library
+  location for preloading, thus not starting on systemd based machines,
+  introduced in |Percona Server| :rn:`5.7.21-20`, was improved to take into
+  account previously created configuration file. Bug fixed :psbug:`3850`.
+
+* A fix introduced in |Percona Server| :rn:`5.7.10-2` which made busy server
+  not to prefer LRU flushing over flush list flushing too strongly was reverted
+  back because of the MT LRU flusher introduction, which removed LRU flushing
+  from the cleaner loop, Bug fixed :psbug:`3812`.
+
+* The possibility of a truncated bitmap file name was fixed in |InnoDB| logging
+  subsystem. Bug fixed :psbug:`3926`.
+
+* Non-instrumented functions were used in cached reads and writes of files in
+  fixed-size units. Bug fixed :psbug:`3937` (upstream :mysqlbug:`90264`).
+
+* A crash in the unsafe query warning checks with views took place for
+  ``UPDATE`` statement in case of statement binlogging format. Bug fixed
+  :psbug:`290`.
+
+MyRocks Changes
+===============
+
+* A re-implemented variable :variable:`rpl_skip_tx_api` allows to turn on simple
+  |RocksDB| write batches functionality, increasing replication performance
+  by the transaction api skip. Bug fixed :jirabug:`MYR-47`.
+
+* Unpack info bytes indicating the trailing space length in a padded varchar
```

Comment:
No user will know what "Unpack info bytes" is
CC @georgelorchpercona

Reply:
Agreed. A simplified/clarified way to describe this issue and fix may read something like 

> Decoding rows with varchar columns under some circumstances can cause assertions and/or data corruption.